docs/references/javascript/clerk/handle-navigation.mdx

```diff
@@ -14,20 +14,20 @@ These are all methods on [the Clerk class](/docs/references/javascript/clerk/cle
 
 ## `handleMagicLinkVerification()`
 
+Completes a magic link verification flow once we've reached the magic link results URL.
+
 ```typescript
 function handleMagicLinkVerification(
     params: HandleMagicLinkVerificationParams,
     customNavigate?: ((to: string) => Promise<unknown>) | undefined
 ): Promise<unknown>;
 ```
 
-Completes a magic link verification flow once we've reached the magic link results URL.
-
 When users click on magic links they get redirected to the URL that was provided during magic link verification flow initialization. The URL will contain a couple of important query parameters added by Clerk. These are called `__clerk_status` and `__clerk_created_session`.
 
 The `__clerk_status` query parameter is the outcome of the verification and can take three values: **verified**, **failed** or **expired**.
 
-The `__clerk_created_session` query parameter will hold the ID of the new session, if one was created as a result of the verification. Since the magic link can be opened at any device and not the one that originated the verification, the new session ID might not be available under [Client.sessions](https://clerk.com/docs/reference/clerkjs/client#sessions).
+The `__clerk_created_session` query parameter will hold the ID of the new session, if one was created as a result of the verification. Since the magic link can be opened at any device and not the one that originated the verification, the new session ID might not be available under [Client.sessions](/docs/references/javascript/client).
```

Comment:
```suggestion
The `__clerk_created_session` query parameter will hold the ID of the new session, if one was created as a result of the verification. Since the magic link can be opened at any device and not the one that originated the verification, the new session ID might not be available under [Client.sessions](/docs/references/javascript/client#properties).
```